.settings/org.eclipse.core.resources.prefs

```diff
@@ -1,4 +1,5 @@
 eclipse.preferences.version=1
 encoding//src/main/java=UTF-8
 encoding//src/test/java=UTF-8
+encoding//src/test/resources=UTF-8
```

Comment:
Review if is needed this config files,lo

Reply:
Changes made locally and will be loaded in the next PR.